bindsnet/analysis/plotting.py

```diff
@@ -399,6 +399,7 @@ def plot_performance(
     performances: Dict[str, List[float]],
     ax: Optional[Axes] = None,
     figsize: Tuple[int, int] = (7, 4),
+    x_scale: int = 1,
 ) -> Axes:
     # language=rst
     """
```

Comment:
Can you add the x_scale parameter to the documentation?

Reply:
done bby